docs/issue-management.md

```diff
@@ -0,0 +1,394 @@
+# Issue Management
+
+This document further describes the developer workflow and how issues are
+managed as introduced in [CONTRIBUTING.md](../CONTRIBUTING.md). Please read
+CONTRIBUTING.md first before proceeding.
+
+- [Issue Management](#issue-management)
+  - [Developer Workflow Overview](#developer-workflow-overview)
+  - [Creating New Issues and PRs](#creating-new-issues-and-prs)
+  - [Good First Issues and Help Wanted](#good-first-issues-and-help-wanted)
+  - [Issue and PR Triage Process](#issue-and-pr-triage-process)
+    - [Issue Triage](#issue-triage)
+    - [PR Triage](#pr-triage)
+  - [Working an Issue](#working-an-issue)
+  - [Issue and PR Labels](#issue-and-pr-labels)
+    - [Issue Kinds](#issue-kinds)
+      - [API Change](#api-change)
+      - [Bug](#bug)
+      - [Cleanup](#cleanup)
+      - [Feature](#feature)
+      - [Deprecation](#deprecation)
+      - [Design](#design)
+      - [Documentation](#documentation)
+      - [Failing Test](#failing-test)
+      - [Support](#support)
+    - [Area](#area)
+    - [Size](#size)
+    - [Triage](#triage)
+    - [Lifecycle](#lifecycle)
+    - [Priority](#priority)
+
+## Developer Workflow Overview
+
+The purpose of this workflow is to formalize a lightweight set of processes that
+will optimize issue triage and management which will lead to better release
+predictability and community responsiveness for support and feature
+enhancements. Additionally, Antrea must prioritize issues to ensure interlock
+alignment and compatibility with other projects including Kubernetes. The
+processes described here will aid in accomplishing these goals.
+
+![developer workflow overview](docs/assets/developer-workflow-opaque-bg.png)
```

Comment:
this does not render properly

Reply:
thanks, updated path to reflect move from CONTRIBUTING.md -> issue-management.md.